lightblue/src/main/java/org/esbtools/eventhandler/lightblue/FindRequests.java

```diff
@@ -25,10 +25,23 @@
 import com.redhat.lightblue.client.request.data.DataFindRequest;
 
 import org.esbtools.eventhandler.lightblue.model.DocumentEventEntity;
+import org.esbtools.lightbluenotificationhook.NotificationEntity;
 
 public abstract class FindRequests {
-    public static DataFindRequest newNotificationsForEntitiesUpTo(String[] entities, int maxEvents) {
-        return null;
+    public static DataFindRequest oldestNewNotificationsForEntitiesUpTo(String[] entities,
```

Comment:
That's fine with me. Technically it should be "unprocessed" notifications. The API has the contract of "don't hand me the same thing more than once" so an implementation of this is to use statuses. This request is one that is modelled around said status -- it doesn't just grab oldest notifications, it is specifically oldest unprocessed notifications.

IMHO it's important to be specific with the request names for readability, but readable to me is not the same as readable to everyone else so, if you think it is okay to be less specific I'm okay with that.